docker/docker-compose.yml

```diff
@@ -71,6 +72,30 @@ services:
       # Disable JWT Auth locally. The JWT_SECRET will be ignored.
       SECURE_CHANNELS: 'false'
       JWT_SECRET: ${JWT_SECRET}
+
+  storage:
+    image: supabase/storage-api:v0.8.3
+    ports:
+      - '5000:5000'
+    depends_on:
+    - db
+    - rest
+    restart: on-failure
+    environment:
+        ANON_KEY: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJzdXBhYmFzZSIsImlhdCI6MTYwMzk2ODgzNCwiZXhwIjoxNjkwNDQzMTM1LCJhdWQiOiIiLCJzdWIiOiIiLCJSb2xlIjoiYW5vbiIsImp0aSI6ImQ1N2ZkMjFlLTNkYjQtNDk3YS1hNGViLTI4ZDY3Yzg2NjEwMiJ9.YySz_BoYxYco0b2aP3iBMt-dVUqC6RpnQiSb4Zt-L-o
+        SERVICE_KEY: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJzdXBhYmFzZSIsImlhdCI6MTYwMzk2ODgzNCwiZXhwIjoxNjkwNDQzMTM1LCJhdWQiOiIiLCJyb2xlIjoic2VydmljZV9yb2xlIiwianRpIjoiOTk4MzJiYmItNDdhNS00NzJhLTgxMDQtYzY4ZGVjZDQyZDVlIn0._DToQdxDcSLsp3tPpacsxswv0p1KRti0QOTmnbfs5JY
+        PROJECT_REF: bjwdssmqcnupljrqypxz # can be any random string
+        POSTGREST_URL: http://rest:3000
+        PGRST_JWT_SECRET: ${JWT_SECRET}
+        DATABASE_URL: postgres://postgres:${POSTGRES_PASSWORD}@db:${POSTGRES_PORT}/postgres
+        PGOPTIONS: "-c search_path=storage"
+        FILE_SIZE_LIMIT: 52428800
+        REGION: us-east-1 # region where your bucket is located
+        GLOBAL_S3_BUCKET: supa-storage-testing  # name of s3 bucket where you want to store objects
```

Comment:
can we remove this if the `STORAGE_BACKEND` is set to `file`?

Reply:
Yup, this needs a backend fix - https://github.com/supabase/storage-api/issues/55 Will get it in